httpclient5-cache/src/main/java/org/apache/hc/client5/http/impl/cache/CacheControlHeaderParser.java

```diff
@@ -162,6 +169,19 @@ public final CacheControl parse(final Header header) {
                 mustRevalidate = true;
             } else if (name.equalsIgnoreCase(HeaderConstants.CACHE_CONTROL_NO_CACHE)) {
                 noCache = true;
+                if (value != null) {
+                    final Tokenizer.Cursor valueCursor = new Tokenizer.Cursor(cursor.getPos() - value.length() -1, buffer.length());
```

Comment:
@ There is a way to parse the value in a single pass without creating an extra `Tokenizer.Cursor`. It is nice to have, though.

Reply:
I made some changes to avoid creating an additional Tokenizer.Cursor, but I'm not completely certain if it's what you're looking for..

Reply:
@arturobernalg My apologies. My suggestion was completely unfounded. Your initial implementation was perfectly fine. I restored it in your commit. Sorry about that.

Reply:
> @arturobernalg My apologies. My suggestion was completely unfounded. Your initial implementation was perfectly fine. I restored it in your commit. Sorry about that.

np @ok2c